### DISCUSSION TITLE
Wondering about the `.pagefind-ui__result-tags` selector in CSS

### DISCUSSION THREAD

**Discussion by nhoizey** [Pagefind/pagefind#569] [Q&A] [answered]
I've seen the `.pagefind-ui__result-tags` selector in the default CSS, but I can't get any search result (on my site or Pagefind's one) with an element class `pagefind-ui__result-tags`.

What's the purpose of this element, and how to get it?

**Comment by bglw** [comment, marked as answer]
If you have custom metadata, these tags are shown below each result. You can see this on the https://xkcd.pagefind.app/ demo site, as the `Date` tag below each result :)

**Reply by nhoizey** [reply]
Oh right, I had not set any metadata apart from `title` so it explains I didn't get anything.
